scripts/plugins/extauth-hook-AD.py

```diff
@@ -177,8 +177,21 @@ def _query_subject(self, sid):
             logger.error("subject %s from sid %s is not valid", subject, sid)
             return None, None
         # CONNAPP\test_user 1  # 1: user, 2: group
-        name, category = comps[0].strip(), comps[1].strip()
-        return name, True if category == "2" else False
+        is_group = True if comps[1].strip() =="2" else False
+        _name = comps[0].strip()
```

Comment:
Why is this prefixed with an underscore?

Reply:
it is name, but not the name wanted, 
let me try to find a `name` for it.